lib/export.js

```diff
@@ -3385,6 +3385,10 @@ class FHIRExporter {
     // Iterate the rules one at a time
     for (const cpr of cp.rules) {
 
+      if (cpr.primaryProfile || cpr.noProfiles) {
+        continue;
```

Comment:
Perhaps I'm misunderstanding, so if you could clarify, that would be helpful. It seems like we are using this function to flag must-support as well as no-profile. At least, by the comment above (which should be updated if it's inaccurate now). If that's the case, then why are we `continue`ing if `cpr.noProfiles`? I think I understand doing so for `cpr.primaryProfile` if this function doesn't deal with that case.

Reply:
That part of the function actually only processes must-support flags. I believe it was built with the idea that future content profile rules would require similar processing, but no-profile and primary-profile work on the level of an Entry, while must-support works on Fields in an entry. So the extra processing is not needed for those flags. The comment above the function is still accurate though, no-profile is handled right at the beginning of the function with the call to `hasContentProfileNoProfile`.